public/app/features/trails/DataTrailsHistory.tsx

```diff
@@ -46,20 +77,9 @@ export class DataTrailHistory extends SceneObjectBase<DataTrailsHistoryState> {
     if (this.state.steps.length === 0) {
       // We always want to ensure in initial 'start' step
       this.addTrailStep(trail, 'start');
-
-      if (trail.state.metric) {
-        // But if our current trail has a metric, we want to remove it and the topScene,
-        // so that the "start" step always displays a metric select screen.
-
-        // So we remove the metric and update the topscene for the "start" step
-        const { metric, ...startState } = trail.state;
-        startState.topScene = getTopSceneFor(undefined);
-        this.state.steps[0].trailState = startState;
-
-        // But must add a secondary step to represent the selection of the metric
-        // for this restored trail state
-        this.addTrailStep(trail, 'metric');
-      }
```

Comment:
@catherineymgui 
Note: This was where reloading from URL would create a start node followed by a metric node.
This isn't necessary now that we have the "Select a new metric" button. The extra node isn't created anymore.